src/schema/__tests__/sale_artwork.test.js

```diff
@@ -90,6 +105,57 @@ describe("SaleArtwork type", () => {
     })
   })
 
+  describe("artwork resolver", () => {
+    it("does not include edition sets when not asking for shallow version", async () => {
+      const query = `
+        {
+          sale_artwork(id: "54c7ed2a7261692bfa910200") {
+            artwork{
+              id
+              edition_sets{
+                id
+              }
+            }
+          }
+        }
+      `
+      expect(await execute(query)).toEqual({
+        sale_artwork: {
+          artwork: {
+            id: "artwork-1",
+            edition_sets: null,
+          },
+        },
+      })
+    })
+    it("does includes edition sets when asking for shallow version", async () => {
```

Comment:
This says "when asking for shallow version" but the code says `shallow: false`. Which is wrong?

Reply:
copy paste + edit went wrong! updating it.

Reply:
oof and the grammar there...